python/hail/table.py

```diff
@@ -1838,7 +1840,7 @@ def flatten(self):
                 z: String
             }]
 
-        with key fields ``a.p`` and ``a.q``.
+        with key ``a.p`` and ``a.q``.
```

Comment:
isn't fields correct here?

Reply:
Yes, I just didn't feel like it added much.  I can put it back if you like.

Reply:
Personally I don't like "key A and B". Key fields is fine, or we should have a notation for key with multiple fields in doc. key ``(a.p, a.q)``? ``[a.p, a.q]``?